game/scripts/vscripts/components/duels/duels.lua

```diff
@@ -331,6 +331,15 @@ function Duels:SavePlayerState (hero)
     mana = hero:GetMana()
   }
 
+  -- If hero is dead during start of the duel, make his saved location his foutain area
+  if hero:IsAlive() == false then
+    if hero:GetTeam() == DOTA_TEAM_GOODGUYS then
+        state.location = Vector(-5221.958496, -139.014923, 387.999023)
```

Comment:
this should use named zones on the map instead, however we'll do that later as a separate unit of work because merging the map is a dick in the dick.